R/caretEnsemble.R

```diff
@@ -142,7 +143,14 @@ predict.caretEnsemble <- function(object, keepNA = TRUE, se = FALSE, return_weig
       message("Predictions being made only for cases with complete data")
     }
     est <- as.numeric(preds %*% object$weights)
-    se.tmp <- apply(preds, 1, FUN = wtd.sd, weights = object$weights, normwt = TRUE)
+    if(dim(preds)[1]<1000){ # use apply for small data
```

Comment:
nitpick: put your comments on their own line.  In this case "# use apply for small data" should come before the code it refers to.